src/utils/requestUtil.ts

```diff
@@ -24,7 +24,7 @@ export async function requestUtil<T>(url: string, credentials?: ServiceClientCre
     if (credentials) {
         await signRequest(requestOptions, credentials);
     }
-    if (method !== "PUT") {
+    if (method !== "PUT" || !body) {
```

Comment:
do you still need method !== "PUT"

Reply:
Also please format the code in this file

Reply:
we may leave the if-clause as this, since this is a bug using "webresource" to send request with body, so we may convert it to nRequest type if we have body to send~